src/rgw/rgw_sal_motr.cc

```diff
@@ -248,9 +255,9 @@ int MotrMetaCache::put(const DoutPrefixProvider *dpp,
   // Inform other rgw instances. Do nothing if it gets some error?
   int rc = distribute_cache(dpp, name, info, UPDATE_OBJ);
   if (rc < 0)
-      ldpp_dout(dpp, 0) <<__func__<<": ERROR: failed to distribute cache for " << name << dendl;
```

Comment:
' ERROR:' will still help in log itself, given it will visually help identify errors faster. log levels atlast are going to be seen in numbers only, which beyond Motr SAL layer wont be consistent. 
retain else where too, where you are removing

Reply:
Agree, `ERROR` should be present in logs.

Reply:
Yes, done
added ERROR string in the log statement